packages/contracts/docs/osx/02-how-to-guides/01-dao/01-best-practices.md

```diff
@@ -2,23 +2,21 @@
 title: Best Practices
 ---
 
-## Advices for Operating your DAO
+## Some Advice When Operating your DAO
 
 ### DOs 👌
 
-- Make sure that at least one address (typically a governance plugin) has `EXECUTE_PERMISSION_ID` permission so that it can execute on the DAO.
-- Check every proposal asking to install, update, or uninstall a plugin with utmost care and review
+- Make sure that at least one address (typically a governance plugin) has `EXECUTE_PERMISSION_ID` permission so that something can be execute on behalf of the DAO.
```

Comment:
```suggestion
- Make sure that at least one address (typically a governance plugin) has `EXECUTE_PERMISSION_ID` permission so that something can be executed on behalf of the DAO.
```